packages/site/src/data/react/podcasts.ts

```diff
@@ -63,7 +63,7 @@ export const podcasts: Podcast<typeof podcastTags[number]>[] = [
 	{
 		title: "React Round Up",
 		image:
-			"https://assets.fireside.fm/file/fireside-images/podcasts/images/4/45ac7cd2-9fff-46c9-842c-2d7e0d34cd4b/cover_small.jpg?v=1",
+			"https://topenddevs.nyc3.digitaloceanspaces.com/ktszo9706o6v5yqqy3s9sor1rgkq?response-content-disposition=inline%3B%20filename%3D%22reactroundup-1-scaled.jpeg%22%3B%20filename%2A%3DUTF-8%27%27reactroundup-1-scaled.jpeg&response-content-type=image%2Fjpeg&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=STO4LZNRL5TU6SLCP4ZY%2F20221006%2Funused%2Fs3%2Faws4_request&X-Amz-Date=20221006T220027Z&X-Amz-Expires=300&X-Amz-SignedHeaders=host&X-Amz-Signature=bb8192309c86957c7578b49c3e1f8afebadb4a744a2ccdc111546a935d16385d",
```

Comment:
I am still seeing a missing image here
<img width="314" alt="Screen Shot 2022-10-07 at 10 54 49 AM" src="https://user-images.githubusercontent.com/67210629/194619959-c859b132-c32a-4245-b953-70ab83d5122e.png">

When I try to go to the new link it says access denied
<img width="645" alt="Screen Shot 2022-10-07 at 10 55 41 AM" src="https://user-images.githubusercontent.com/67210629/194620108-1449e32f-264f-475a-94df-c255715481d4.png">

Reply:
Looks like they implemented this after my PR :/ Fixing it!